crypto/asymmetric/keypair.go

```diff
@@ -71,6 +71,12 @@ func (k *PublicKey) toECDSA() *ecdsa.PublicKey {
 
 // Serialize is a function that converts a public key
 // to uncompressed byte array
+//
+// TODO(leventeliu): use SerializeUncompressed, which is about 40 times faster than
+// SerializeCompressed.
```

Comment:
65 bytes vs 33 bytes, but 40 times faster on Unserialization.
@xq262144 @zeqing-guo how you guys think?